op_crates/fetch/26_fetch.js

```diff
@@ -990,7 +990,7 @@
         this.url = new URL(String(input)).href;
       }
 
-      if (init && "method" in init) {
+      if (init && "method" in init && init.method) {
```

Comment:
Shouldn't it be if (init && init.method) ?

Reply:
Ideally we can just check `init.method`, I'm just following existing code like https://github.com/denoland/deno/blob/3fc4df3887d1611aeb87b8d1fe08b227b7e724a9/op_crates/fetch/26_fetch.js#L998-L1000